templates/ts-apollo-postgres-backend/doc.graphql

```diff
@@ -0,0 +1,147 @@
+fragment NoteFields on Note {
```

Comment:
This should not be generated here and be reverted

Reply:
Sorry, this was me doing manual testing